studio/components/to-be-cleaned/Storage/Storage.types.ts

```diff
@@ -3,3 +3,14 @@ import { PolicyFormField } from 'components/interfaces/Auth/Policies/Policies.ty
 export interface StoragePolicyFormField extends PolicyFormField {
   allowedOperations: string[]
 }
+
+export interface StorageBucket {
+  id: string
+  name: string
+  owner: string
+  public: boolean
+  file_size_limit: number
+  allowed_mime_types: string[]
+  created_at: string
+  updated_at: string
+}
```

Comment:
Is there any reason this seems to be duplicated from `studio/components/interfaces/Storage/Storage.types.ts`?